module/mempool/herocache/dns_cache.go

```diff
@@ -16,12 +17,23 @@ type DNSCache struct {
 	txtCache *stdmap.Backend
 }
 
-func NewDNSCache(sizeLimit uint32, logger zerolog.Logger) *DNSCache {
+func NewDNSCache(sizeLimit uint32, logger zerolog.Logger, ipCollector module.HeroCacheMetrics, txtCollector module.HeroCacheMetrics) *DNSCache {
 	return &DNSCache{
-		txtCache: stdmap.NewBackendWithBackData(
-			herocache.NewCache(sizeLimit, 8, heropool.LRUEjection, logger)),
-		ipCache: stdmap.NewBackendWithBackData(
-			herocache.NewCache(sizeLimit, 8, heropool.LRUEjection, logger)),
+		txtCache: stdmap.NewBackend(
+			stdmap.WithBackData(
+				herocache.NewCache(
+					sizeLimit,
+					herocache.DefaultOversizeFactor,
+					heropool.LRUEjection,
+					logger,
+					txtCollector))),
+		ipCache: stdmap.NewBackend(
+			stdmap.WithBackData(
+				herocache.NewCache(sizeLimit,
```

Comment:
Any reason why `sizeLimit` is not on its own line?

Reply:
fixed https://github.com/onflow/flow-go/pull/2093/commits/15f1b2ec9914a42214993228d49bdbc41c000cb1